COMP 120 Tinkering Graphics-JH182233/Pictures/Snow

```diff
@@ -0,0 +1,73 @@
+"""
+ Animating multiple objects using a list.
+ Sample Python/Pygame Programs
+ Simpson College Computer Science
+ http://programarcadegames.com/
+ http://simpson.edu/computer-science/
+ 
+ Explanation video: http://youtu.be/Gkhz3FuhGoI
+"""
+ 
+# Import a library of functions called 'pygame'
+import pyglet
+import random
+ 
+# Initialize the game engine
+pygame.init()
+ 
+BLACK = [0, 0, 0]
+WHITE = [255, 255, 255]
+ 
+# Set the height and width of the screen
+SIZE = [400, 400]
+ 
+screen = pygame.display.set_mode(SIZE)
+pygame.display.set_caption("Snow Animation")
+ 
+# Create an empty array
+snow_list = []
+ 
+# Loop 50 times and add a snow flake in a random x,y position
+for i in range(50):
+    x = random.randrange(0, 400)
+    y = random.randrange(0, 400)
+    snow_list.append([x, y])
+ 
+clock = pygame.time.Clock()
+ 
+# Loop until the user clicks the close button.
+done = False
+while not done:
+ 
+    for event in pygame.event.get():   # User did something
+        if event.type == pygame.QUIT:  # If user clicked close
+            done = True   # Flag that we are done so we exit this loop
```

Comment:
Could you have indented the comment in this line to be at the same level of indentation as the previous two lines?